Woof/WoofTests/ViewModel/SitterProfileViewModelTests/SitterProfileViewModelSaveTests.swift

```diff
@@ -0,0 +1,38 @@
+import XCTest
+
+final class SitterProfileViewModelSaveTests: XCTestCase {
+    func testSaveMethodExistsInAPI() {
+        SitterProfileViewModel().save()
+    }
+
+    func testSaveMethodSuccessfullySavedModifiedSitterProperties() {
+        let viewModel = SitterProfileViewModel()
+
+        let newName = Sitter.Test.johnSmith.name
+        let newSurname = Sitter.Test.johnSmith.surname
+        let newPhone = Sitter.Test.johnSmith.phone
+        let newBio = Sitter.Test.johnSmith.bio
+        let newPricePerHour = Sitter.Test.johnSmith.pricePerHour
+
+        viewModel.name = newName
+        viewModel.surname = newSurname
+        viewModel.phone = newPhone
+        viewModel.bio = newBio
+        viewModel.pricePerHour = String(newPricePerHour)
+        viewModel.save()
+
+        guard let data = KeyValueStorage(KeyValueStorage.Name.currentSitter)
+            .loadData(for: KeyValueStorage.Key.currentSitter) else {
+            return
+        }
+        guard let currentSitter = try? JSONDecoder().decode(Sitter.self, from: data) else {
+            return
+        }
```

Comment:
```suggestion
// Given

        let viewModel = SitterProfileViewModel()

        let newName = Sitter.Test.johnSmith.name
        let newSurname = Sitter.Test.johnSmith.surname
        let newPhone = Sitter.Test.johnSmith.phone
        let newBio = Sitter.Test.johnSmith.bio
        let newPricePerHour = Sitter.Test.johnSmith.pricePerHour

        viewModel.name = newName
        viewModel.surname = newSurname
        viewModel.phone = newPhone
        viewModel.bio = newBio
        viewModel.pricePerHour = String(newPricePerHour)
        
        // When

        viewModel.save()

        // Then

        guard let data = KeyValueStorage(KeyValueStorage.Name.currentSitter)
            .loadData(for: KeyValueStorage.Key.currentSitter) else {
            XCTFail("Unexpected storage error")
            return
        }
        guard let currentSitter = try? JSONDecoder().decode(Sitter.self, from: data) else {
            XCTFail("Unexpected decoding error")
            return
        }
```
Please, use `Given/When/Then` format and add `XCTFail`.

Reply:
Check the rest tests.

Reply:
Fixed it.

Reply:
What about `XCTFail` for `else` branches of `guard` statements?

Reply:
My bad, fixed it.